### PR TITLE
Fix parsing vic-machine output for docker params [full ci]

### PR DESCRIPTION
**Pull request by hmahmood**

### CODE REVIEW

File: tests/resources/VCH-Util.robot [hunk 1]
```diff
@@ -64,7 +64,7 @@ Get Docker Params
     Remove Environment Variable  DOCKER_HOST  DOCKER_TLS_VERIFY  DOCKER_CERT_PATH
 
     # Split the log log into pieces, discarding the initial log decoration, and assign to env vars
```
Comment (emlin) [inline tests/resources/VCH-Util.robot]: "log log"?

Comment (emlin) [inline tests/resources/VCH-Util.robot]: Might have a simple description for why need to split from right?

**Comment by emlin** [comment]
cosmetic comments. lgtm

File: tests/resources/VCH-Util.robot [hunk 2]
```diff
@@ -64,7 +64,7 @@ Get Docker Params
...
-    ${logmon}  ${logday}  ${logyear}  ${logtime}  ${loglevel}  ${vars}=  Split String From Right  ${line}  ${SPACE}  5
+    ${logmon}  ${logday}  ${logyear}  ${logtime}  ${loglevel}  ${vars}=  Split String  ${line}  ${SPACE}  5
```
Comment (anchal-agrawal) [inline tests/resources/VCH-Util.robot]: `Group11-Upgrade` tests are failing in build 7667, but I'm not sure if those failures are related to this.

**Pull request merged**